### PR TITLE
Clarify changelog note

### PR DESCRIPTION
**Pull request by amyblais**
For https://community.mattermost.com/private-core/pl/b7ykqipf8pn47xu7xu5ue5k8uw

### CODE REVIEW

**Comment by ewwollesen** [comment]
Should we add an additional note that this should only be enabled if users are experiencing performance issues?

**Comment by amyblais** [comment]
> Should we add an additional note that this should only be enabled if users are experiencing performance issues?

Added now 👍

File: source/deploy/mattermost-changelog.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ See [this walkthrough video](https://www.youtube.com/watch?v=udC2OCTGooc&feature
 #### User Interface (UI)
  - Improved readability by displaying system messages on multiple lines when editing a channel header.
  - Combined "joined/left" event types in system messages.
- - Added a new user preference to disable webapp prefetching via **Settings > Advanced > Allow Mattermost to prefetch channel posts**.
+ - Added a new user preference to disable webapp prefetching via **Settings > Advanced > Allow Mattermost to prefetch channel posts**. This setting should only be enabled if users are experiencing performance issues. Client Performance Debugging needs to be enabled in the System Console in order for this setting to appear.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:42]:
```suggestion
 - Added a new user preference to disable webapp prefetching via **Settings > Advanced > Allow Mattermost to prefetch channel posts**. This setting should only be enabled if users are experiencing performance issues. You must enable **Client Performance Debugging** in the System Console by going to **Environment > Developer** in order for this setting to appear.
```

**Review by cwarnermm — APPROVED** [review]
Provided feedback inline. Approving.

**Review by ewwollesen — APPROVED** [review]

**Review by BenCookie95 — CHANGES_REQUESTED** [review]

**Comment by BenCookie95** [comment]
@cwarnermm Could you look over the wording again please? We changed it around a bit

**Review by BenCookie95 — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]
Great update! Clear, actionable guidance. Well done.

**Pull request merged**